Airports/EGTU/GEO

```diff
@@ -0,0 +1,25 @@
+; Start Dunkeswell (EGTU) GEO
+EGTU Dunkeswell     N050.51.36.000 W003.14.05.000
```

Comment:
Not sure where this single coordinate has come from? Have a look at the following file to have a look how this should be formatted: https://github.com/VATSIM-UK/UK-Sector-File/blob/master/Airports/EGKK/SMR/Geo.txt